sql/2019/05_Third_Parties/05_06.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Top 100 third party domains by request volume
+SELECT
+  thirdPartyDomain,
+  COUNT(*) AS totalRequests,
+  SUM(requestBytes) AS totalBytes
+FROM (
+  SELECT
+      SAFE_CAST(REGEXP_EXTRACT(payload, r'_bytesIn":(\d+)') AS INT64) AS requestBytes,
+      NET.HOST(url) AS requestDomain,
+      DomainsOver50Table.requestDomain as thirdPartyDomain
+    FROM
+      `httparchive.requests.2019_07_01_mobile`
```

Comment:
Do you also want to include desktop in your analysis?

Reply:
I was anticipating doing everything on mobile for consistency because some metrics are only available on lighthouse dataset which is mobile.